packages/playground/worker/newUrl/classic-worker.js

```diff
@@ -1,4 +1,4 @@
-importScripts('/classic.js')
+importScripts(`${self.location.origin}/classic.js`)
```

Comment:
if the worker type is `classic` and inline worker use base64 format. there must got a error.

Reply:
only module worker and `worker format: iife` support inline worker? 

### classicWorker

build and run to got error. for `importScripts('/classic.js')`

### format: es

dynamic import in worker can't find the web route.